web/src/features/stations/components/WxStationDropdown.tsx

```diff
@@ -37,18 +42,23 @@ const WxStationDropdown = (props: Props) => {
   const { stations, stationsByCode, error: errorFetchingStations } = useSelector(
     selectStations
   )
-  let isThereInvalidCode = false
+
+  let isThereUnknownCode = false
   const maxNumOfSelect = props.maxNumOfSelect || 3
-  const autocompleteValue = props.stationCodes.map(code => {
-    const station = stationsByCode[code]
-    if (station) {
-      return station
+  const autocompleteValue: Option[] = props.stationCodes.map(code => {
+    const s = stationsByCode[code]
```

Comment:
Why rename from station to s? I would think station is better?

Reply:
Okay will update it